refresh/runner.go

```diff
@@ -17,7 +18,10 @@ func (m *Manager) runner() {
 			// kill the previous command
 			pid := cmd.Process.Pid
 			m.Logger.Success("Stopping: PID %d", pid)
-			cmd.Process.Kill()
+			if err := cmd.Process.Signal(syscall.SIGINT); err != nil {
```

Comment:
@bketelsen do you still have a windows box? If so, could you try this PR on it to make sure it works?